src/entrypoint.sh

```diff
@@ -9,7 +9,7 @@ invoke () {
     then
         /usr/local/bin/invoke $@
     else
-        /usr/local/bin/invoke $@ >> /usr/src/{{project_name}}/invoke.log 2>&1
+        /usr/local/bin/invoke $@ >> /usr/src/geonode_proj/celery.log 2>&1
```

Comment:
`geonode_proj` should be reverted to `{{project_name}}`

Reply:
Done

Reply:
@jkariukidev `celery.log` should be reverted to `invoke.log`. This log is not specific to celery.
No need to touch parts that are not related to the tasks of this issue....